src/main/java/seedu/address/model/person/Socials.java

```diff
@@ -1,4 +1,42 @@
 package seedu.address.model.person;
 
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
 public class Socials {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Socials should not be blank and must be alphanumeric, with the exception of the underscore symbol: \"_\".";
+
+    /*
+     * The first character of the social media handle must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\w]*";
```

Comment:
Validation Regex is not changed?

Reply:
Maybe we can remove the constraint? Just make sure theres no white space as the first character.